### PR TITLE
grapl-plugin-sdk.analyzer - your analyzer is now a grpc service, that consumes GraphQueryService

### PR DESCRIPTION
**Pull request by wimax-grapl**
### Which issue does this PR correspond to?
https://github.com/grapl-security/issue-tracker/issues/918

### What changes does this PR make to Grapl? Why?
Introduce a new grapl-analyzerlib - this will eventually be renamed to the old one, but I want to do the replace-and-rip-out as a separate PR. 

Introduce an example `suspicious_svchost` analyzer built on the new analyzerlib.

### How were these changes tested?
1. I created a concrete `suspicious_svchost.exe`
2. an analyzer is deployed and guaranteed to be healthy in `test_deploy_suspicious_svchost_analyzer` - but we don't really pass messages through it yet. That will be tested soon.

### CODE REVIEW

File: src/python/grapl-analyzerlib-new/example_analyzers/suspicious_svchost.py
Comment (wimax-grapl) [inline src/python/grapl-analyzerlib-new/example_analyzers/suspicious_svchost.py:80]: @colin-grapl what's this about lol

File: src/python/grapl-analyzerlib-new/example_analyzers/suspicious_svchost.py
Comment (wimax-grapl) [inline src/python/grapl-analyzerlib-new/example_analyzers/suspicious_svchost.py:102]: IMO we should get the service_config inside of `serve_analyzer()` instead of passing it in; but writing an example analyzer here helps expose weird API contracts like this

File: src/python/python-proto/python_proto/api/graph_query/v1beta1/client.py
Comment (wimax-grapl) [inline src/python/python-proto/python_proto/api/graph_query/v1beta1/client.py:30]: this was a bad pattern; these `python-proto` clients should be as bone-simple-maps-to-proto as possible, zero smarts

File: src/python/python-proto/python_proto/api/plugin_sdk/analyzers/v1beta1/server.py
Comment (wimax-grapl) [inline src/python/python-proto/python_proto/api/plugin_sdk/analyzers/v1beta1/server.py:69]: I'm intentionally keeping this here for the time being; the reflection proved useful for testing, until the rest of analyzer infra is hooked up, and then I'll remove it.

File: src/rust/plugin-registry/tests/test_deploy_plugin.rs
Comment (wimax-grapl) [inline src/rust/plugin-registry/tests/test_deploy_plugin.rs:132]: (I have another branch where I rebased atop analyzer-exectuion-sidecar and un-ignore this. I'm honestly down to just remove this test entirely instead of partially committing it here if that's preferable to people?)

**Review by jgrillo-grapl — CHANGES_REQUESTED** [review]
Can we hold off on merging this until Colin's PR containing my analyzer-execution-sidecar changes lands? That way it'll be "in-order" and we'll actually be able to write tests for this code.

EDIT: reviewed the code and LGTM once the test runs

**Comment by wimax-grapl** [comment]
stalebot dont you dare

File: src/python/grapl-plugin-sdk/example_analyzers/suspicious_svchost.py
Comment (wimax-grapl) [inline src/python/grapl-plugin-sdk/example_analyzers/suspicious_svchost.py:106]: this is just me spitballing about the API we expose to the customers

**Review by jgrillo-grapl — APPROVED** [review]

**Pull request merged**
